### ISSUE TITLE
Unable to automatically scroll/move  and  find an element, which is not in view by findElementByAccessibilityId() 

### ISSUE THREAD

**Issue by angad16** [microsoft/WinAppDriver#1039]
I am trying to automate Microsoft Store and while doing so , i want to scroll to an element which is currently not in view , i tried using findElementByAccessibilityId of an app and neither able to find it nor it got scrolled.
Error :
org.openqa.selenium.NoSuchElementException: An element could not be located on the page using the given search parameter

  if (driver.findElementsByName("Maximize Microsoft Store").size() > 0) {
            driver.findElementByAccessibilityId("Maximize").click();
       }
      
        WebElement ele = driver.findElementByAccessibilityId("9PB2MZ1ZMB1S"); // this is the id of itunes app which is not in view 

Can you please help me ?

**Comment by anunay1** [comment]
which version of win app driver are u using

**Comment by angad16** [comment]
Hi @anunay1 ,
i am using v1.1

**Comment by anunay1** [comment]
trying adding some delay before you try to click

**Comment by angad16** [comment]
it is working the same @anunay1

**Comment by naeemakram** [comment]
What type of application are you testing? UWP, WinForms, or WPF?
